specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json

```diff
@@ -1398,6 +1398,11 @@
           "description": "Expression evaluated.",
           "type": "string"
         },
+        "expressionKind": {
```

Comment:
This was an oversight in the previous swagger spec.  The readonly property has been part of the API response from the time the api-version was introduced.